### DISCUSSION TITLE
Browser Extension  |  Knowledge discovery/creation

### DISCUSSION THREAD

**Discussion by arkRedM** [athensresearch/athens#888] [General]
I think knowledge lives in three steps 
1. Creation | Discovery
2. Application and augmentation
3. Management

to solve the first part -- Discovery -- I think most of it is internet research where you jump through pages and take notes -- and eventually copy to Athens 

So Athens-buddy(the name is subject to change) will be a chrome extension and before starting your research just click on the icon and add title, initial notes, or labels(or not) and set a time limit(optional) and go to any page after that and
1. start highlighting text -- to make a note of it
2. rt-click(or some better semantic -- maybe drag-drop?) images/videos to copy links 
each highlight can have labels previously added attached to it (a generic way to organize)

After the session, one-click copy, and paste in Athens as
- Link
   - note 1 #label1 #label2
   - note 2 #label3
all URLs formatted; maybe links to pages that are already created.

Would love any feedback on this.

**Comment by lswank** [comment]
Some thoughts:

- Capture the title of each page.￼
- Give firefox users some love￼
- Don't capture the URL if nothing was marked or saved

There is a real danger of duplicating work. Allow me to play devils advocate for a moment while not discouraging you. I actually want you to build this thing. But it's important for us to think through the thing and see if there is a shortcut or a better way to what you want to do. 

￼￼￼Why not just build a connector for ￼￼Hypothes.is? (I started out a year ago building one with￼ Stian for Roam.) <No image capture, yes.>

Why not ask Jeremy Ho who built Eloquent? It does all of the text capture you want, just not automatically. <No FF extension. No image capture.>

**Reply by phrohdoh** [reply]
> Give firefox users some love

Perhaps https://github.com/abhijithvijayan/web-extension-starter (or similar) can help with this.

**Reply by arkRedM** [reply]
On Hypothesis 
It's a lovely idea. But it strays off from the current target of seamless data capture and UX is not optimized for it. It's a great starting point. Love the collaborative idea.
On eloquent
I was unable to signup. ping to eloquent works isn't working. From the videos and ss it's really close to what I am thinking right now. Have to use it personally to have more ideas from there.

**Comment by sawhney17** [comment]
@arkRedM  correct me if i'm wrong, Roam Highlighter does this i think and is compaitable with athnes

**Reply by arkRedM** [reply]
they should be compatible(it's just text at the end of the day)

**Comment by jefftangx** [comment]
There's at least 5 Roam extensions I've heard of. Might be helpful to compile a list of them to centralize the best parts, if that list doesn't already exist!

**Reply by sawhney17** [reply]
I'll create a page for this on the roam db
